charts/alfresco-search-enterprise/templates/liveindexing-deployment.yaml

```
@@ -24,14 +25,14 @@ spec:
         {{- include (printf "alfresco-search-enterprise.%s.labels" $serviceName) $ | nindent 8 }}
     spec:
       serviceAccountName: {{ include "alfresco-search-enterprise.serviceAccountName" $ }}
-      {{- include "alfresco-common.component-pod-security-context" .Values | indent 4 }}
+      {{- include "alfresco-common.component-pod-security-context" $.Values | indent 4 }}
       {{- include "alfresco-common.imagePullSecrets" $ | indent 6 }}
       containers:
         - name: {{ include (printf "alfresco-search-enterprise.%s.name" $serviceName) $ }}
-          image: "{{ index $.Values "liveIndexing" (printf "%s" $serviceName)  "image" "repository" }}:\
-            {{ index $.Values "liveIndexing" (printf "%s" $serviceName)  "image" "tag" }}"
-          imagePullPolicy: {{ index $.Values "liveIndexing" (printf "%s" $serviceName)  "image" "pullPolicy" }}
-            {{- include "alfresco-common.component-security-context" .Values | indent 8 }}
+          image: "{{ index $.Values "liveIndexing" $serviceName "image" "repository" }}:\
+            {{ index $.Values "liveIndexing" $serviceName "image" "tag" }}"
+          imagePullPolicy: {{ index $.Values "liveIndexing" $serviceName "image" "pullPolicy" }}
+            {{- include "alfresco-common.component-security-context" $.Values | indent 8 }}
```

If we define `$service`, then why not using it?

```suggestion
          image: {{$service.image.repository }}:{{ $service.image.tag }}
          imagePullPolicy: {{ $service.image.pullPolicy }}
            {{- include "alfresco-common.component-security-context" $.Values | indent 8 }}```